changeme.py

```diff
@@ -603,7 +610,7 @@ def build_target_list(targets, creds, name, category):
             fp = Fingerprint(c['name'], fp=c['fingerprint'])
 
             for path in fp.urls:
-                url = '%s://%s:%s%s' % (proto, target, port, path)
```

Comment:
Not sure why the change here, but only having three format strings and four arguments causes a TypeError.